Service/OptimizeImageService.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Magenerds
+ * @package    Magenerds_OptimizeMedia
+ * @author     Mahmood Dhia <m.dhia@techdivision.com>
+ * @copyright  Copyright (c) 2018 TechDivision GmbH (http://www.techdivision.com)
+ * @link       http://www.techdivision.com/
+ */
+
+namespace Magenerds\OptimizeImage\Service;
+
+use Magenerds\OptimizeImage\Api\Service\OptimizeImageServiceInterface;
+use Magenerds\OptimizeMedia\Helper\OptimizeImage as OptimizeImageHelper;
+
+class OptimizeImageService implements OptimizeImageServiceInterface
+{
+    /**
+     * @var OptimizeImageHelper
+     */
+    private $optimizeImage;
+
+    /**
+     * CrawlService constructor.
+     * @param OptimizeImageHelper $optimizeImage
+     */
+    public function __construct(OptimizeImageHelper $optimizeImage)
+    {
+        $this->optimizeImage = $optimizeImage;
+    }
+
+    /**
+     * Optimize Image
```

Comment:
Please use `@inheritdoc` so it can use it from the interface.

Reply:
done